outputs.tf

```diff
@@ -27,13 +37,13 @@ output "rai_policy_id" {
 }
 
 output "resource" {
-  description = "The cognitive account resource created."
+  description = "The cognitive account resource created in AzureRM schema."
   value       = local.resource_block
```

Comment:
We should not output the entire resource any more, we should use anti-corruption layer

Reply:
This `local.resource_block` is implemented as anti-corruption layer, it's composed with several local expressions and won't be affected by any incoming changes automatically in the future.